### ISSUE TITLE
When I use nni Compressor Pruner, a 'NOT compressed'  error is occurred.

### ISSUE THREAD

**Issue by yeliang2258** [microsoft/nni#2029]
The error is ：
![image](https://user-images.githubusercontent.com/30516196/74147729-cc731980-4c3e-11ea-8368-37e930f1d180.png)

My code is：

import math
import torch
import torch.nn as nn
import torch.nn.functional as F
from torchvision import datasets, transforms
from nni.compression.torch import L1FilterPruner
from vgg import VGG

def main():
    torch.manual_seed(0)

    model = VGG(depth=16)

    torch.save(model.state_dict(), 'vgg16_cifar10.pth')

    # Pruning Configuration, in paper 'PRUNING FILTERS FOR EFFICIENT CONVNETS',
    # Conv_1, Conv_8, Conv_9, Conv_10, Conv_11, Conv_12 are pruned with 50% sparsity, as 'VGG-16-pruned-A'
    configure_list = [{
        'sparsity': 0.5,
        'op_types': ['default'],
        'op_names': ['feature.0', 'feature.24', 'feature.27', 'feature.30', 'feature.34', 'feature.37']
    }]

    # Prune model and test accuracy without fine tuning.
    print('=' * 10 + 'Test on the pruned model before fine tune' + '=' * 10)
    pruner = L1FilterPruner(model, configure_list)
    pruned_model = pruner.compress()
    pruner.export_model(model_path='test_pruned_model.pth', mask_path='test_mask_model.pth')
    mask_state_dict = torch.load('./test_mask_model.pth')
    print(mask_state_dict)

if __name__ == '__main__':
    main()

The code is from the nni project example. I tried multiple pruning algorithms, but all got the same error.

**Comment by Cjkkkk** [comment]
Hi @yeliang2258, could you provide the nni version you are using, also could you try update nni to the latest version v1.4 and see if error still exists?

**Comment by yeliang2258** [comment]
> Hi @yeliang2258, could you provide the nni version you are using, also could you try update nni to the latest version v1.4 and see if error still exists?

At first I tried version v1.3 and found the above problem, then I installed the latest version using source code and found that it didn't work either

**Comment by Cjkkkk** [comment]
You have to train to calculate mask, default mask is None, which is why you get `not compressed` message.

**Comment by yeliang2258** [comment]
> You have to train to calculate mask, default mask is None, which is why you get `not compressed` message.

I get it, thanks!
